README.md

```diff
@@ -126,6 +146,13 @@ If a functionality is missing, it is easy to call it using `apply()` or `apply_(
 ```python
 tensordict_uniform = tensordict.apply(lambda tensor: tensor.uniform_())
 ```
+
+``apply()`` can also be great to filter a tensordict, for instance:
+```python
+data = TensorDict({"a": 1.0, "b": 1}, [])
+data_float = data.apply(lambda x: x if x.dtype == torch.float else None) # contains only the "a" key
```

Comment:
nit: this example may need more explanation as the user could expect the values of `data` to be float / int and not tensors.

Reply:
yep i'm packing many things there, let's make it more straightforward